app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -745,5 +720,142 @@ export default {
     font-size: 29px;
     line-height: 40px;
   }
+
+  .text_dont_just_play_code img{
+    max-width: 410px;
+    margin: 0 0 0 auto;
+  }
+
+  .video-iframe-section {
+    justify-content: flex-start;
+    z-index: 0;
+    margin-top: -120px;
+  }
+
+  .leaderboard-component {
+    color: #000;
+  }
+
+  .text-2021-section {
+    width: 100%;
+    overflow-x: hidden;
+    display: flex;
+    justify-content: flex-end;
+    margin-bottom: 120px;
+  }
+
+  .clan-btn-section {
+    margin-bottom: 300px;
+  }
+
+  .text-are-you-an-educator {
+    justify-content: flex-start;
+    margin-top: 100px;
+  }
+  .text-code-section {
+    width: 100%;
+    overflow-x: hidden;
+  }
+  .esports-flyer-optimized-section {
+    margin-bottom: 100px;
+  }
+  .esports-flyer-optimized-section img {
+    transform: translateY(100px);
+  }
+
+  @media screen and (min-width: 768px) {
+    .btn-primary.btn-moon, .blazing-battle {
+      padding: 20px 100px;
+    }
+    .btn-primary.btn-moon {
+      background-color: #d1b147;
+      border-radius: 4px;
+      color: #232323;
+      text-shadow: unset;
+      text-transform: uppercase;
+      font-weight: bold;
+      letter-spacing: 0.71px;
+      line-height: 24px;
+      font-size: 18px;
+
+      &:hover {
+        background-color: #f7d047;
+        transition: background-color .35s;
+      }
```

Comment:
Do we need two separate styles for the button at two different media queries? Instead we could have a `.btn-primary.btn-moon` outside any media queries with all the shared behavior.
Currently on smaller screens the button becomes very small and the hover behavior is lost.


I think the button should still have padding on smaller screens to provide a larger finger press surface.

![image](https://user-images.githubusercontent.com/15080861/103310785-46ce9880-49cd-11eb-8742-ec73796daaf7.png)

Reply:
Put `btn-primary.btn-moon` outside of media queries and added more padding in button.

 
![Competitive-AI-coding-eSports-from-CodeCombat-9](https://user-images.githubusercontent.com/75598717/103356062-ed05c580-4ad5-11eb-8e13-44686edde623.png)